.github/workflows/test.yml

```diff
@@ -46,3 +46,18 @@ jobs:
       shell: cmd
       run: |
         cd tests && .\test-prog-x64.exe
+
+  tests_on_alpine_x86_64:
+    name: Tests on Alpine (x86_64 and i686)
```

Comment:
Could you remove `and i686`? It isn't tested here.

Reply:
Mmm right! By the way, can you approve the CI to run?